DNN Platform/Library/Common/Utilities/HtmlUtils.cs

```diff
@@ -49,7 +49,7 @@ public class HtmlUtils
         private static readonly Regex HtmlDetectionRegex = new Regex("<(.*\\s*)>", RegexOptions.Compiled);
         private static readonly Regex StripWhiteSpaceRegex = new Regex("\\s+", RegexOptions.Compiled);
         private static readonly Regex StripNonWordRegex = new Regex("\\W*", RegexOptions.Compiled);
-        private static readonly Regex StripTagsRegex = new Regex("<[^<>]*>", RegexOptions.Compiled);
+        private static readonly Regex StripTagsRegex = new Regex("<.*?>", RegexOptions.Compiled | RegexOptions.Multiline);
```

Comment:
this regex can't match the string which have line break:
![20160330165459](https://cloud.githubusercontent.com/assets/3364448/14136979/353abd7a-f698-11e5-972d-6f4fa9d011a4.png)